Update DataAPI_change_log.md (#3188)
SkylineCommunications/dataminer-docs: release-notes/DxMs/DataAPI_change_log.md

> [!NOTE]
>

> > - The *config* endpoint can only be used when an auto-generated connector exists and which is identified by the mandatory **type** HTTP header field.

Reviewer: ```suggestion
> - The *config* endpoint can only be used when an auto-generated connector exists that is identified by the mandatory **type** HTTP header field.
```
